src/h5web/visualizations/line/MappedLineVis.tsx

```diff
@@ -47,9 +48,7 @@ function MappedLineVis(props: Props): ReactElement {
     scaleType
   );
 
-  const abscissaLabel = mapperState && axesLabels[mapperState.indexOf('x')];
-  const abscissas = abscissaLabel && axesValues[abscissaLabel];
-  assertOptionalArray<number>(abscissas);
```

Comment:
The `assert` is no longer needed as it was done beforehand in the hook `useNxAxisParams`.